src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/AutoCrafterListener.java

```diff
@@ -17,6 +21,8 @@
 import io.github.thebusybiscuit.slimefun4.implementation.items.electric.gadgets.Multimeter;
 import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
 
+import static org.bukkit.Bukkit.getRecipesFor;
```

Comment:
Check [this](https://github.com/Slimefun/Slimefun4/blob/master/CONTRIBUTING.md#1-imports). Static imports are not allowed. Why you can't import `Bukkit` class?

Reply:
I don't know why... But it just highlights my code wrong if I just import the `Bukkit` class.

Reply:
It only works either when it's a static import, or I write out the full directory when using the method.

Reply:
Hmm... What program do you use for editing java code?

Reply:
I use IntelliJ.

Reply:
Well. Try to rebuild project or something that.

Reply:
Oh! NVM sorry I just figured it out ~

Reply:
okay